src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass_LocalFunctions.cs

```diff
@@ -52,10 +54,16 @@ protected TLocalFunctionState GetOrCreateLocalFuncUsages(LocalFunctionSymbol loc
             {
                 usages = CreateLocalFunctionState(localFunc);
                 _localFuncVarUsages[localFunc] = usages;
+                LocalFuncUsageCount++;
             }
             return usages;
         }
 
+        protected bool HasLocalFuncUsage(LocalFunctionSymbol localFunc)
```

Comment:
>HasLocalFuncUsage [](http://example.com/codeflow?start=23&length=17)

The name feels somewhat confusing. The function tells us if we already allocated a state for a local function. It doesn't really tells us if the function was called or used in some different way.